MoreLinq/Slice.cs

```diff
@@ -51,6 +51,11 @@ public static IEnumerable<T> Slice<T>(this IEnumerable<T> sequence, int startInd
                 ? SliceImpl(asList, startIndex, count)
                 : SliceImpl(sequence, startIndex, count);
         }
+        
+        public static IEnumerable<T> Slice<T>(this IEnumerable<T> sequence, int startIndex)
+        {
+            return sequence.Slice(startIndex, sequence.Count - startIndex);
```

Comment:
Missing argument null checks, and `IEnumerable` does not have a `Count` property. Moreover, just invoking `Count()` would iterate the sequence twice.